index.html

```diff
@@ -5,43 +5,95 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Awesome Books</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+			integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
     <link rel="stylesheet" href="./style/style.css">
+    <script src="https://cdn.jsdelivr.net/npm/luxon@2.0.2/build/global/luxon.min.js"></script>
   </head>
   <body>
-    <div class="container">
-      <table class="table table-striped mt-4">
-        <thead>
-          <tr>
-            <th>Author</th>
-            <th>Title</th>
-            <th># of pages</th>
-            <th>Delete</th>
-          </tr>
-        </thead>
-        <tbody id="book-list">
-    
-        </tbody>
-      </table>
-      <h1 class="ml-auto mt-5">Awesome Books</h1>
-      <div class="form_box">
-        <form id="book-form">
-          <div class="form-group">
-            <label for="title">Title</label>
-            <input type="text" id="title" class="form-control">
-          </div>
-          <div class="form-group">
-            <label for="author">Author</label>
-            <input type="text" id="author" class="form-control">
-          </div>
-          <div class="form-group">
-            <label for="nPages">Pages</label>
-            <input type="text"  id="nPages" class="form-control">
-          </div>
-            <input type="submit" value="Add Book" class="btn btn-primary btn-block " >
+    <nav class="navbar navbar-expand-lg  bg-white border-secondary border mt-2 ">
+			<div class="container-fluid">
+				<a class="navbar-brand home" href="#">Awesome Books</a>
+				<button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarTogglerDemo03"
+					aria-controls="navbarTogglerDemo03" aria-expanded="false" aria-label="Toggle navigation">
+					<span class="navbar-toggler-icon"></span>
+				</button>
+				<div class="collapse navbar-collapse" id="navbarTogglerDemo03">
+					<ul class="navbar-nav ms-auto">
+						<li class="nav-item ">
+							<a class="nav-link list-button border-end" href="#">
+								Book List
+							</a>
+						</li>
+						<li class="nav-item">
+							<a class="nav-link add-new border-end" href="#">
+								New Book
+							</a>
+						</li>
+						<li class="nav-item add-contacts">
+							<a class="nav-link" href="#">

```

Comment:
- [ ] OPTIONAL
- To enhance the user experience of this design kindly ensure that an active menu link is highlighted(color change or change in opacity). In other words, the current active page should have the corresponding menu item highlighted. We really want our designs to as much as possible mimic the provided [wireframe template](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md).

**Your design menu:**
![image](https://user-images.githubusercontent.com/12745474/129426047-097c31fa-a0eb-4e14-8d3a-2dc692293eba.png)

**The wireframe menu design:**
![image](https://user-images.githubusercontent.com/12745474/129183812-a30acaa9-4f6c-4d4e-973b-5dd83a591c8d.png)